transaction_test.go

```diff
@@ -742,7 +742,7 @@ func TestSubmitTransaction_EnvelopeSignature(t *testing.T) {
 		result, err := b.ExecuteNextTransaction()
 		assert.NoError(t, err)
 
-		assert.True(t, fvmerrors.IsAccountNotFoundError(result.Error))
+		assert.ErrorContains(t, result.Error, "account not found")
```

Comment:
does it make sense to check that the err code is present instead?

Reply:
IsAccountNotFoundError internally calls HasErrorCode, which right now only checks the error code in the first error (1101) and ignore the error codes in the other errors when fvm returns multiple errors.  Hence the failure.  we can probably update HasErrorCode to check for secondary error codes as well